src/media-utils.ts

```diff
@@ -0,0 +1,125 @@
+import { PureCloudWebrtcSdk } from './client';
+import { log } from './logging';
+import { LogLevels, SdkErrorTypes } from './types/enums';
+import browserama from 'browserama';
+import { throwSdkError } from './utils';
+
+const PC_AUDIO_EL_CLASS = '__pc-webrtc-inbound';
+let _hasTransceiverFunctionality: boolean | null = null;
+
+declare var window: {
```

Comment:
Could this be a `const`?

Reply:
no, this is typescript for saying "this already exists in this format", not an actual declaration.